streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamThreadTest.java

```diff
@@ -932,9 +929,8 @@ public void shouldShutdownTaskManagerOnClose() {
             internalTopologyBuilder,
             CLIENT_ID,
             new LogContext(""),
-            new AtomicInteger(),
-            new AtomicLong(Long.MAX_VALUE)
-        ).updateThreadMetadata(getSharedAdminClientId(CLIENT_ID));
+            new AtomicLong(Long.MAX_VALUE),
+            new AtomicInteger()).updateThreadMetadata(getSharedAdminClientId(CLIENT_ID));
```

Comment:
Also here it should be:
```suggestion
            new AtomicInteger()
        ).updateThreadMetadata(getSharedAdminClientId(CLIENT_ID));
```

Reply:
foxed the 5 of these as well